### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by plesiecki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,14 @@ More information about available helpers can be found on
 
 To register a custom helper use [@HandlebarsHelper](src/main/java/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelper.java) annotation.
 
+### Custom cache template
+
+Set handlebars template cache by bean of type [TemplateCache](https://github.com/jknack/handlebars.java/blob/master/handlebars/src/main/java/com/github/jknack/handlebars/io/TemplateLoader.java)
```
Comment (platan) [inline README.md]: TemplateCache links to TemplateLoader.java. Is it OK?

File: README.md [hunk 2]
```diff
@@ -43,6 +43,14 @@ More information about available helpers can be found on
...
+
+### Custom template loader
+
+Set handlebars template loader by bean of type [TemplateLoader](https://github.com/jknack/handlebars.java/blob/master/handlebars/src/main/java/com/github/jknack/handlebars/cache/TemplateCache.java)
```
Comment (platan) [inline README.md]: TemplateLoader links to TemplateCache.java. Is it OK?

File: README.md (hunk 1 above)
Comment (plesiecki) [inline README.md]: ;)

File: src/main/java/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelpersAutoConfiguration.java [hunk 3]
```diff
@@ -117,6 +118,19 @@ public void registerHelpers() {
...
+    @Configuration
+    @ConditionalOnClass(StringHelpers.class)
+    static class StringHelpersAutoConfiguration {
+
+        @Autowired
+        private HandlebarsViewResolver handlebarsViewResolver;
+
+        @PostConstruct
+        public void registerHelpers() {
+            StringHelpers.register(handlebarsViewResolver.getHandlebars());
```
Comment (platan) [inline src/main/java/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelpersAutoConfiguration.java:130]: Do we have to register it on our own? It is a [built-in helper](https://github.com/jknack/handlebars.java#built-in-helpers).

Comment (plesiecki) [inline src/main/java/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelpersAutoConfiguration.java:130]: Yes, we do. It isn't registered by default.

**Comment by platan** [comment]
Now everything looks great!

**Pull request merged**